web/modules/custom/rdf_entity/src/RdfPermissions.php

```diff
@@ -41,19 +41,25 @@ public function rdfTypePermissions() {
    */
   protected function buildPermissions(RdfEntityType $type) {
     $type_id = $type->id();
-    $type_params = array('%type_name' => $type->label());
+    $type_params = ['%type_name' => $type->label()];
 
-    return array(
-      "create $type_id rdf entity" => array(
+    return [
+      "create $type_id rdf entity" => [
```

Comment:
This is an example of how OG deals with the Node permissions (from `OgEventSubscriber::provideDefaultNodePermissions()`:

```
      $permission_values = [
        [
          'name' => "create $bundle_id content",
          'title' => $this->t('%type_name: Create new content', $args),
          'operation' => 'create',
        ],
```

So for RDF entities this should be:

```
      $permission_values = [
        [
          'name' => "create $type_id rdf entity",
          'title' => $this->t('Create new %type_name RDF entity', $args),
          'operation' => 'create',
        ],
```